src/api/migrations/20190624064739-create-test-articl.js

```diff
@@ -0,0 +1,55 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Articles', {
```

Comment:
@Ramadhan0 why do you need this file?

Reply:
I need to have Article table, this is because for a user to be an Author he/she needs to have an article